WordPress/src/main/java/org/wordpress/android/ui/sitecreation/segments/NewSiteCreationSegmentsFragment.kt

```diff
@@ -0,0 +1,72 @@
+package org.wordpress.android.ui.sitecreation.segments
+
+import android.os.Bundle
+import android.os.Parcelable
+import android.support.annotation.LayoutRes
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.ViewGroup
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.sitecreation.NewSiteCreationBaseFormFragment
+import org.wordpress.android.ui.sitecreation.NewSiteCreationListener
+import org.wordpress.android.util.DisplayUtils
+import org.wordpress.android.widgets.RecyclerItemDecoration
+
+class NewSiteCreationSegmentsFragment : NewSiteCreationBaseFormFragment<NewSiteCreationListener>() {
+    private var linearLayoutManager: LinearLayoutManager? = null
+    private val keyListState = "list_state"
```

Comment:
>Sorry for missing this in my previous review, I think we can make this a top level const.

No worries! Fixed in the UI branch.

>Also, just to follow up on our discussions yesterday, I found that the PagedListAdapter does keep the scroll position intact on rotation. I'd be curious to see how easy it'd be to use that even for something simple like this. Not saying we should, but I'd at least think about the pros and cons of it.

Interesting idea! I'm a bit worried it might confuse us in the future, as we'd be looking for pagination on that screen even though there wouldn't be any. Imho it's an overkill for something so simple. Or do you see any significant benefits?

Reply:
> I'm a bit worried it might confuse us in the future, as we'd be looking for pagination on that screen even though there wouldn't be any.

Technically, there'd still be pagination. The pagination in `PagedListAdapter` doesn't necessarily mean that it happens on the remote, it could just be the DB pagination.

> Imho it's an overkill for something so simple. Or do you see any significant benefits?

Yes definitely overkill right now. There are some benefits such as preserving the current state during rotation, but nothing significant enough for now. Just brought it up as an off point for now, so we can keep it in mind.

Reply:
👍